README.md

```diff
@@ -21,4 +131,111 @@ Things you may want to cover:
 
 * Deployment instructions
 
-* ...
+### Setup
+
+- Download the Zip file or Clone the repo with:
+   - git clone https://github.com/karam084/budget-app
+   - To access the project directory
+   - cd budget-app
+
+### Install
+
+ - bundle install
+ - rails db:migrate
+
+### Usage
+
+To run the project, execute the following command:
+
+- rails s 
+- Check http://localhost:3000/ in your browser
+
+### Deployment
+
+Run the following command to install all dependencies.
+
+$ bundle install
+$ code .
+
+## Database
+To create your local Database
+
+$ rails db:create
+$ rails db:migrate
+
+### Run tests
+To install rspec, in the terminal kindly run this command
+
+$ gem install rspec
+To run tests, please run this command
+
+$ rspec ./spec/#{filename}_spec.rb 
+To run all tests, please run this command
+
+or $ spec  
+
+<!-- AUTHORS -->
+
+## 👥 Authors <a name="authors"></a>
```

Comment:
- [ ] OPOTIONAL
I think you should change the header title _Authors_ to _Author_ in your ReadMe, as this project is worked on by a sole contributor.

Reply:
Solved, Thank you